MD/md-4/rollover-gas.md

```diff
@@ -0,0 +1,52 @@
+# Rollover Gas
+The current suggestion for the implementation of MCR is to offset the gas costs for the last attester. This would be done transparently, by rewarding the last attester for their work in additional compensation for this gas cost. This would be a simple solution to the problem of disincentivizing the last attester. However, this solution is naive as it may incentivize an attempt to be the last attester--introducing an incentive for latency to the chain.
+
+## Base Model
+We model the utility of the offset as follows...
+
+```math
+\begin{align}
+&U_{n} = -E[G_{\mu}] \\
+&U_{l} = \delta(|\mathcal{V}|)U_{o} - E[G_{\mathcal{V}}] \\
+\end{align}
+```
+$U_{n}$ is the utility of any attester that is not last.
+
+$U_{l}$ is the utility of the last attester.
+
+$E [.] is the expectation value of a variable.$
+
+$E[G]$ is the expected gas cost modeled as a random variable $G$.
+
+$\delta(|\mathcal{V}|)$ is reward rate determined by the number of validators in the set $\mathcal{V}$.
```

Comment:
need better definitions and explanations: 

- V is the complete set of validators?
- why is the reward rate determined by the number of validators